src/lostcity/cache/FontType.ts

```diff
@@ -131,4 +131,42 @@ export default class FontType {
 
 		return size;
     }
+
+    split(str: string, maxWidth: number): string[] {
+        if (str.length === 0) {
+            // special case for empty string
+            return [str];
+        }
+
+        let lines: string[] = [];
```

Comment:
```suggestion
        const lines: string[] = [];
```

Doesn't appear to be reassigned (can do `.push` to a `const` no problem)